### DISCUSSION TITLE
Moving to GitHub Discussions

### DISCUSSION THREAD

**Discussion by arikfr** [getredash/redash#5913] [Announcements]
Hello everyone,

We have decided to move our community discussions from the [Discourse forum](https://discuss.redash.io/) to GitHub Discussions. This is part of our effort to simplify and bring everything into one place.

We hope that this change will make it easier for everyone to participate in the community discussions and share their ideas.

As a result of this change, the Discourse forum will move into read-only mode. If anyone wants to help with creating an archive from it or importing here, please reach out.

Thank you for your understanding and we hope you find the new platform useful!

**Comment by Jengro777** [comment]
I hope that redash can launch a small new version as soon as possible, so that we can have a new starting point

**Reply by arikfr** [reply]
Can you clarify what do you mean by small version?

**Reply by Jengro777** [reply]
> Can you clarify what do you mean by small version?

https://github.com/getredash/redash/discussions/5962#discussioncomment-5822866

Everyone is currently unable to merge the code into Redash

**Reply by Jengro777** [reply]
> Can you clarify what do you mean by small version?

If this continues, everyone's enthusiasm for participation will decrease; Or there may be new forks to continue with the work. But everyone is still looking forward to your participation

**Reply by rpmcoding** [reply]
@arikfr please can you reply at https://github.com/getredash/redash/discussions/5962.  Everyone is keen to get going here but needs more access.
